python/pyarrow/_s3fs.pyx

```diff
@@ -87,6 +87,29 @@ def resolve_s3_region(bucket):
 
     return frombytes(c_region)
 
+class AwsRetryStrategy(object):
```

Comment:
Maybe 'S3RetryStrategy' to be in line with the C++ side? And then 'AwsStandardS3RetryStrategy'

Reply:
```suggestion
class AwsRetryStrategy:
```